node/src/chain_spec.rs

```diff
@@ -619,6 +619,7 @@ pub fn bridge_test_config() -> ChainSpec {
     )
 }
 
+#[allow(unused_variables)]
```

Comment:
Why not just remove unused vars?

Reply:
because some of them used in rococo and alphanet features